src/main/java/com/powsybl/openloadflow/util/ParameterConstants.java

```diff
@@ -28,6 +28,9 @@
     public static final String LOW_IMPEDANCE_BRANCH_MODE_PARAM_NAME = "lowImpedanceBranchMode";
     public static final LowImpedanceBranchMode LOW_IMPEDANCE_BRANCH_MODE_DEFAULT_VALUE = LowImpedanceBranchMode.REPLACE_BY_ZERO_IMPEDANCE_LINE;
 
+    public static final String POWER_FACTOR_CONSTANT_PARAM_NAME = "powerFactorConstant";
```

Comment:
Do not forget to increase documentation. Our web site is here: https://github.com/powsybl/powsybl.github.io

Reply:
Did you find the related page? it's that one: pages/documentation/simulation/powerflow/openlf.md